src/components/AvailabilityStatusLabel/messages.ts

```diff
@@ -3,14 +3,14 @@ import { defineMessages } from "react-intl";
 export const messages = defineMessages({
   published: {
     defaultMessage: "Published on {date}",
-    description: "product publication date"
+    description: "{type} publication date"
```

Comment:
It's not templated so translator will see this string exactly as you've written it

Reply:
We can't do it like this. If we reuse the same message for different objects, there's no way for translators to differentiate between gendered nouns ("produkt opublikowany" vs. "wersja opublikowana").

Also, make sure `{date}` is actually a date and not date&time ("opublikowany dnia 12:31 w czwartek").